index.html

```diff
@@ -5,9 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 <body>
-    <header></header>
-    <section></section>
+    <header>
+        <div class="status-bar"></div>
+        <div class="toolbar">
+            <div class="logo-placeholder">
+                <p class="name white-color">Thomas Obuya</p>
```

Comment:
- [ ] I think it will be better if you make your website semantic by wrapping your logo in an anchor `<a>` tag such that it is clickable, which will be easier for users to navigate the homepage from another page which is a good user experience. Kindly consider that 👍🏽